components/widget/match/summary/widget_match_summary_casters.lua

```diff
@@ -0,0 +1,61 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Widget/Match/Summary/Casters
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Flags = require('Module:Flags')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+
+local Widget = Lua.import('Module:Widget')
+local HtmlWidgets = Lua.import('Module:Widget/Html/All')
+local Div = HtmlWidgets.Div
+local Link = Lua.import('Module:Widget/Basic/Link')
+
+---@class MatchSummaryCasters: Widget
+---@operator call(table): MatchSummaryCasters
+local MatchSummaryCasters = Class.new(Widget)
+
+---@return Widget?
+function MatchSummaryCasters:render()
+	if type(self.props.casters) ~= 'table' then
+		return nil
+	end
+
+	local casters = Array.map(self.props.casters, function(caster)
+		if Logic.isEmpty(caster) then
+			return nil
+		end
```

Comment:
is this even needed?
casters should be an array, right?

Reply:
Existing solution did this check, I changed it now to checking for caster.name exists (as the display will be f'd without it)